other/docker/windows/build_toxcore.sh

```diff
@@ -123,6 +145,11 @@ build()
 
     sed -i "s|^prefix=.*|prefix=${RESULT_PREFIX_DIR}|g" ${RESULT_PREFIX_DIR}/lib/pkgconfig/*.pc
     sed -i "s|^libdir=.*|libdir=${RESULT_PREFIX_DIR}/lib|g" ${RESULT_PREFIX_DIR}/lib/*.la
+
+    # copy all deps headers to result directory
+    if [ "${CROSS_COMPILE}" = "false" ]; then
+        cp "${DEP_PREFIX_DIR}"/* "${RESULT_PREFIX_DIR}" -R
+    fi
```

Comment:
We shouldn't provide headers of dependencies, just those of tox.

Reply:
But we provide them [here](https://build.tox.chat/job/libtoxcore-toktok_build_windows_x86-64_static_release/lastSuccessfulBuild/artifact/). What's the difference?

Reply:
Because it also provides libopus, libsodium, libvpx, but here we just provide libtoxcore.

Reply:
Done.